### PR TITLE
BUGFIX Fix Neos.Ui version information in build script

### PR DESCRIPTION
**Pull request by mhsdesign**
Resolves: #3781

it will show "UI version: 9.0.3" for tagged builds from neos-ui-compiled and "UI version: 9.0-dev" when requiring dev-9.0 of neos-ui-compiled

### CODE REVIEW

File: Build/Jenkins/release-neos-ui.sh [hunk 1]
```diff
@@ -44,7 +44,7 @@ make install
 # acutal release process
 
 # build
-make build-production
+make build-subpackages
```
Comment (Sebobo) [inline Build/Jenkins/release-neos-ui.sh:47]: why is this necessary?

Comment (mhsdesign) [reply Build/Jenkins/release-neos-ui.sh:47]: simplifies stuff ... was and leftover artefact ... we dont care of the build here and just want to release the npm stuff

**Review by grebaldi — APPROVED** [review]
LGTM :+1:

**Review by Sebobo — APPROVED** [review]
